scripts/create_pcap_dir

```diff
@@ -2,7 +2,7 @@
 
 set +x
 
-if [[ "$PCAP_GCS_FUSE" == true && "${PCAP_FSN_ENABLED}" == true ]]; then
+if [[ "$PCAP_GCS_EXPORT" == true && "${PCAP_FSN_ENABLED}" == true ]]; then
     # only create the PCAP files directory in GCS if FUSE and FSN are both enabled
```

Comment:
[NIT]: only create the PCAP files directory in GCS if GCS export and FSN are both enabled

Reply:
looks like you updated this before I published my comment